src/azul/service/manifest_service.py

```diff
@@ -2029,17 +2037,19 @@ class ReplicaKeys:
         or the replica's entity ID.
         """
         hub_id: str
-        replica_id: str
+        replica_ids: list[str]
 
     def _replica_keys(self) -> Iterable[ReplicaKeys]:
-        hub_type = self.implicit_hub_type
         request = self._create_request()
         for hit in request.scan():
-            replica_id = one(hit['contents'][hub_type])['document_id']
-            if self.entity_type != hub_type:
-                replica_id = one(replica_id)
+            document_ids = [
+                document_id
+                for entity_type in self.hot_entity_types
+                for inner_entity in hit['contents'].to_dict().get(entity_type, ())
```

Comment:
```suggestion
                for inner_entity in hit['contents'].to_dict().get(entity_type, ())
```

Why is to_dict() used here?

Reply:
Because the ES library's `AttrDict` class does not support the `get` method